### ISSUE TITLE
users_json does not work for lux ad mosam

### ISSUE THREAD

**Issue by wilco375** [csvalpha/sofia#908]
Since `users_json` and `api_token` in `users_controller.rb` use specific API endpoints, they do not work for non-amber hosts such as the website of lux ad mosam

**Comment by lodewiges** [comment]
has this been done with the alpha oath plugin?

**Comment by wilco375** [comment]
> has this been done with the alpha oath plugin?

No, it is not implemented yet. It only synchronizes individual users when they log in, but it is currently not possible to sync all users.

**Comment by DrumsnChocolate** [comment]
how much prio does this have?
